src/elm/Data/Currency.elm

```diff
@@ -10,6 +11,8 @@ module Data.Currency
         , zero
         )
 
+import Numeral
```

Comment:
Should we be only exposing the things we are using? i.e. ```import Numeral exposing (format)```

Reply:
The way it is right now, none of the functions in `Numeral` are imported into this namespace meaning that all functions will need to be prefixed by the module name e.g. `Numeral.format`.

`import Numeral` is expose _none_
`import Numeral exposing (..)` is expose _all_